### PR TITLE
delete export_genome_features_protein_to_fasta from spec and recompile

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

File: GenomeFileUtil.spec [hunk 1]
```diff
@@ -236,9 +236,6 @@ module GenomeFileUtil {
     funcdef export_genome_as_gff(ExportParams params)
                 returns (ExportOutput output) authentication required;
 
-    funcdef export_genome_features_protein_to_fasta(ExportParams params)
```
Comment (Tianhao-Gu) [inline GenomeFileUtil.spec:239]: I assume no other model is using this function? We should probably mention this in RELEASE note.

Comment (MrCreosote) [reply GenomeFileUtil.spec:239]: They can't be, the return var is `output` which is never initialized.

+1 on release notes though

Comment (Xiangs18) [reply GenomeFileUtil.spec:239]: updated release notes 👍

File: kbase.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.11.7
+    0.11.8
```
Comment (MrCreosote) [inline kbase.yml:12]: 0.11.7 isn't registered anywhere, there's no need for a version bump

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.8] - 2024-05-23
+export_genome_features_protein_to_fasta was removed from GenomeFileUtil.spec
```
Comment (MrCreosote) [inline RELEASE_NOTES.md:9]:
```suggestion
Unusable `export_genome_features_protein_to_fasta` function was removed
```

File: kbase.yml (hunk 2 above)
Comment (Xiangs18) [reply kbase.yml]: 👍

Comment (MrCreosote) [reply kbase.yml]: Release notes still have a version bump

Comment (Xiangs18) [reply kbase.yml]: so I just add under [0.11.7]? But how do I update the corresponding date?

Comment (MrCreosote) [reply kbase.yml]: yep. I'd just set the date to TBD for now then change it when you register

Comment (Xiangs18) [reply kbase.yml]: 👍

**Review by MrCreosote — COMMENTED** [review]
LGTM but again @Tianhao-Gu should approve

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**